EvalView/templates/EvalView/data-assessment.html

```diff
@@ -245,7 +245,7 @@
             <div class="col-sm-3">
                 <p>
                 <label class="radio-inline">
-                    <input type="radio" name="rank" id="radio-{{ loop.index }}" value="{{ rank_value }}">
+                    <input type="radio" name="rank" id="radio-{{ forloop.counter }}" value="{{ rank_value }}">
```

Comment:
Thanks for finding this. Was this causing collection of incorrect scores? I don't recall issues here from my testing and previous campaigns, so I hope not.